rubrics/services/src/main/resources/application.properties

```diff
@@ -0,0 +1,189 @@
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].description=
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[0].title=Inadequate
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[0].points=0
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[0].description=
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[1].title=Poor
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[1].points=5
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[1].description=
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[2].title=Fair
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[2].points=10
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[2].description=
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[3].title=Good
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[3].points=15
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[3].description=
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[4].title=Exceptional
+rubrics.instance.layouts[flex].language[en-US].default-rubric.criterions[1].ratings[4].points=20
+
+#
+# Default criterion used when an individual new criterion is added in a flex layout
+rubrics.instance.layouts[flex].language[en-US].default-criterion.title=New Criterion
+rubrics.instance.layouts[flex].language[en-US].default-criterion.description=
+rubrics.instance.layouts[flex].language[en-US].default-criterion.ratings[0].title=Inadequate
+rubrics.instance.layouts[flex].language[en-US].default-criterion.ratings[0].points=0
+rubrics.instance.layouts[flex].language[en-US].default-criterion.ratings[0].description=
+rubrics.instance.layouts[flex].language[en-US].default-criterion.ratings[1].title=Meets expectations
+rubrics.instance.layouts[flex].language[en-US].default-criterion.ratings[1].points=1
+rubrics.instance.layouts[flex].language[en-US].default-criterion.ratings[1].description=
+rubrics.instance.layouts[flex].language[en-US].default-criterion.ratings[2].title=Exceeds expectations
+rubrics.instance.layouts[flex].language[en-US].default-criterion.ratings[2].points=2
+rubrics.instance.layouts[flex].language[en-US].default-criterion.ratings[2].description=
+
+#
+# Default rating used when an individual new rating is added in a flex layout
+rubrics.instance.layouts[flex].language[en-US].default-rating.title=New Rating
+rubrics.instance.layouts[flex].language[en-US].default-rating.description=New rating description
+rubrics.instance.layouts[flex].language[en-US].default-rating.points=0
+
+
+
+rubrics.instance.layouts[flex].language[es-ES].default-rubric.title=Nueva R\u00FAbrica
```

Comment:
Spring Boot is pretty flexible, these can be separated into discrete property files per language and we can come up with a mechanism to optionally add them to the Spring Boot Environment. It is not using standard ResourceBundle support for these, as they are default data mapped into model types, but I think we can make adjustments to accommodate the common working pattern for translators.